docs/podman-pause.1.md

```diff
@@ -9,11 +9,28 @@ podman\-pause - Pause one or more containers
 ## DESCRIPTION
```

Comment:
The synopsis for each of these man pages needs to be updated.  the "CONTAINER ..." now needs to be in brackets two as it's not always required due to the --all switch.  See https://github.com/containers/buildah/blob/master/docs/buildah-rm.md

Reply:
fixed